.env.example

```
@@ -12,3 +12,7 @@ APP_TIMEOUT=5s
 
 GOOGLE_SERVICE_ACCOUNT={"type":"service_account", "project_id": ""}
 GOOGLE_BILL_BUCKET_NAME=your-bucket-name
+
+VALKEY_ADDR=http://localhost:8080
+VALKEY_PASSWORD=your-valkey-password
+VALKEY_DB=your-valkey-db
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VALKEY_DB key should go before the VALKEY_PASSWORD key

(UnorderedKey)